aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -254,9 +256,15 @@ Resources:
           ResourceRecords: [!GetAtt <%=daemon%>.PublicIp]
 <%   end -%>
 <% end -%>
+
+<%  if environment == :production -%>
```

Comment:
I think it’s worth adding the `www.` redirect logic to all environments that use CloudFront (`cdn_enabled`). That would enable to us to potentially add an integration or browser test that exercises the `www.` redirect as well. I think we’d insert that logic in this block?

Reply:
plus then we get to test outside of prod!